pythonStacker/plotHistogramsRedo.py

```diff
@@ -31,8 +31,12 @@ def parse_arguments():
 
     parser.add_argument("--EFT_ratio", dest="EFT_ratio", action="store_true", default=False)
     parser.add_argument("--EFT_fullbkg", dest="EFT_fullbkg", action="store_true", default=False)
+    parser.add_argument("--BSM_ratio", dest="BSM_ratio", action="store_true", default=False)
+    parser.add_argument("--BSM_fullbkg", dest="BSM_fullbkg", action="store_true", default=False)
     parser.add_argument("--SBRatio", dest="SBRatio", action="store_true", default=False)
     parser.add_argument("--shapes", dest="shapes", action="store_true", default=False)
+    parser.add_argument("--hasBSM", action="store_true", default=True)
```

Comment:
Is this flag needed?

Reply:
I use it to specify the output subdirectory in order not to have too many plots in the same one, not strictly necessary but useful for organization reasons

Reply:
ok but can't the existing args.UseBSM be used instead? just to remove additional arguments that don't really do anything